### ISSUE TITLE
Lobby stuck on Disable Inspector Recording

### ISSUE THREAD

**Issue by asperlea** [Protowalker/OWDataExtractor#1]
I may be doing something wrong, but when I try to copy the data at the end of rounds, I'm getting no variables existing. It seems to be because Disable Inspector Recording is set after the Setup phase, which causes all future actions be ignored. I tried removing that action, but now I'm seeing some buggy behavior with the end of round event. Does this code still work in recent patches?

**Comment by Protowalker** [comment]
It should still work, but it's buggy. I'm working on a new version but real life has gotten in the way.

For your specific issue, have you waited for the entire game to finish? I believe I changed the behavior recently to only be at the total end but I can revert that or show you how.

**Comment by asperlea** [comment]
I did try waiting for the entire game to finish as well. I may try to debug this weekend, thanks for replying! If I sort it out is it cool if I submit a PR?

**Comment by Protowalker** [comment]
Of course; that would be great! I'll gladly merge any changes you make. Right now there's a second version in progress and because I didn't understanding git at the time it's in a folder called "taketwo." Feel free to check that out as well.

**Comment by asperlea** [comment]
Oh gotcha ok yeah I was wondering what that folder was for :)
